### PR TITLE
feat(cluster linking): Add Cluster Linking feature to Premium

### PR DESCRIPTION
**Pull request by Meggielqk**

### CODE REVIEW

File: en_US/cluster_linking/cluster_linking.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+## Get Started with Cluster Linking
+
+To create the cluster linking between two Premium deployments, you need to set up the cluster linking feature in both deployments. This section demonstrates how to create cluster linking between two Premium deployments. 
+
+Before you start, you need to create two deployments in different regions. For example, you can create two deployments with names as `deployment-us` and `deployment-eu`. For more information on how to create Premium deployments, see [Create a Premium Deployment](../create/premium.md).
+
+### Set Up the Network
+
+To ensure secure and efficient inter-cluster connectivity, the clusters should communicate with each other exclusively through private network connections. You need to set up [NAT Gateway](../vas/nat-gateway.md) for both deployments and the NAT gateway status must be running.
```
Comment (0721Betty) [inline en_US/cluster_linking/cluster_linking.md:29]: To ensure secure and efficient inter-cluster connectivity, the clusters should communicate with each other exclusively through private network connections -> In order for the two deployments to be linked to each other ?
@zibuyu1995 Looking forward to your comments

File: en_US/cluster_linking/cluster_linking.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+To ensure secure and efficient inter-cluster connectivity, the clusters should communicate with each other exclusively through private network connections. You need to set up [NAT Gateway](../vas/nat-gateway.md) for both deployments and the NAT gateway status must be running. 
+
+### Create Cluster Linking in `deployment-us`
+
+1. Click the deployment card of `deployment-us` in your Console.
+
+2. Select **Cluster Linking** from the left navigation menu.
+
+3. Click **New** at the upper right corner. On the **New Cluster Linking** page, configure the following options:
+   - **Deployment Name**: Select the deployment name of the Premium deployment that you want to link to. In this demonstration, it is `deployment-eu`. You can find the deployment name information on the deployment **Settings** page.
```
Comment (0721Betty) [inline en_US/cluster_linking/cluster_linking.md:38]: "You can find the deployment name information on the deployment **Settings** page." can be deleted

File: en_US/cluster_linking/cluster_linking.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+   
+   - **Address**: The MQTT host and port of the deployment to be linked. It is automatically filled when you select the deployment name.
+   
+   - **Username**: Enter the username for authentication to the `deployment-eu`, if it is configured on its Authentication page.
+   
+   - **Password**: Enter the password for authentication to the `deployment-eu`, if it is configured on its Authentication page.
+   
+   - **Client ID Prefix**: Specify a prefix for Client IDs used by MQTT connections to the `deployment-eu`, for example, `from-eu`.
```
Comment (0721Betty) [inline en_US/cluster_linking/cluster_linking.md:46]: deployment-us

File: en_US/cluster_linking/cluster_linking.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+   
+     Depending on the cluster size and configuration, multiple MQTT client connections may be established to the remote cluster, and each client must have a unique ClientID. You can control how these ClientIDs are allocated by setting the *Client ID Prefix* for these connections.
+   
+     :::
+   
+   - **Topics**: List of MQTT topic filters that specify which messages the local cluster will receive from the remote cluster, for example, `/from-eu`. You can click the plus icon to add more topics.
+   
+     ::: tip
+   
+     The local cluster expects to receive messages published to these topics from the remote cluster. This list can be empty, meaning the local cluster will not receive any messages from the remote cluster if no topics are specified.
```
Comment (0721Betty) [inline en_US/cluster_linking/cluster_linking.md:58]: local cluster -> current deployment
remote cluster -> another deployment

File: en_US/cluster_linking/cluster_linking.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+### Create Cluster Linking in `deployment-eu`
+
+1. Click the deployment card of `deployment-eu` in your Console.
+2. Select **Cluster Linking** from the left navigation menu.
+3. Click **New** at the upper right corner. On the **New Cluster Linking** page, configure the following options:
+   - **Deployment Name**: Select `deployment-us` from the drop-down list.
+   - **Address**: The MQTT host and port of the `deployment-us`. It is automatically filled when you select the deployment name.
+   - **Username**: Enter the username for authentication to the `deployment-us`, if it is configured on its Authentication page.
+   - **Password**: Enter the password for authentication to the `deployment-us`, if it is configured on its Authentication page.
+   - **Client ID Prefix**: Specify a prefix for Client IDs used by MQTT connections to the remote cluster. For example, `from-us`.
```
Comment (0721Betty) [inline en_US/cluster_linking/cluster_linking.md:77]: remote cluster -> `deployment-us`

from-us -> from-eu

File: en_US/cluster_linking/cluster_linking.md (hunk 3 above)
Comment (Meggielqk) [reply en_US/cluster_linking/cluster_linking.md:46]: 我觉得前面应该还是 `deployment-eu`吧，后面那个应该是 `from-us`。这句的意思是说“为用来连接到 `deployment-eu` 的 MQTT 连接所使用的客户端 ID 设置前缀。”

Comment (0721Betty) [reply en_US/cluster_linking/cluster_linking.md]: ok

**Review by 0721Betty — APPROVED** [review]

**Review by zibuyu1995 — APPROVED** [review]

**Pull request merged**